package.json

```diff
@@ -165,6 +166,8 @@
     "puppeteer": "^10.2.0",
     "rollup": "^2.50.6",
     "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-shim": "^1.0.0",
```

Comment:
I didn't notice where this gets used

Reply:
`build-lr` package.json script

Reply:
oh! `shim`. yea, the other PR uses it. removing here.